nifi-commons/nifi-expression-language/pom.xml

```diff
@@ -93,5 +93,15 @@
             <artifactId>commons-text</artifactId>
             <version>1.8</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.8.1</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.13</version>
```

Comment:
can we go to 1.14

Reply:
Bumped and re-tested. Similarly, AFAICT nothing in the commons-codec release notes suggest a problem.